pkg/database/object/object.go

```diff
@@ -0,0 +1,44 @@
+package object
```

Comment:
An overall comment on this: is there any reason why you are centralizing all of your database logic in one place like this? It seems like you're duplicating a lot of what you can get from gorm out of the box. 

In my experience, it makes more sense to pass a `gorm.db` pointer around after you create it upon startup to your models, so they can use gorm directly and you don't have to write all of these queries manually.

Let me know if this doesn't make sense, I can try and explain it better.